# Class: Bug::Test_ExceptionDE
**Inherits:** Test::Unit::TestCase
    




#Instance Methods
## test_cleanup_data_error() [](#method-i-test_cleanup_data_error)

